tests/api/test_create_issue.py

```diff
@@ -0,0 +1,47 @@
+import random
+import string
+
+import allure
+import pytest
+import requests
+from requests.auth import HTTPBasicAuth
+
+from data import const, fixtures
+
+
+class TestCreateIssue:
+    ISSUES_ENDPOINT = const.JIRA_URL + '/rest/api/2/issue'
+    EMPTY_SUMMARY_ERROR = "You must specify a summary of the issue."
+    LONG_SUMMARY_ERROR = "Summary must be less than 255 characters."
+    LONG_SUMMARY = "This is very long summary which cannot be randomly generated due to the parallel execution +" \
+                   "This is very long summary which cannot be randomly generated due to the parallel execution +" \
+                   "This is very long summary which cannot be randomly generated due to the parallel execution +" \
+                   "This is very long summary which cannot be randomly generated due to the parallel execution +"
+    issue_id: str = None
+
+    @pytest.mark.api
+    @allure.title("Create issue - positive")
+    def test_create_issue_positive(self):
+        issue_json = fixtures.get_issue_json(const.PREFIX + str(round(random.random() * 100000)))
+        headers = {'Content-Type': 'application/json'}
```

Comment:
I know you can, please, split the test logic from util code.

headers = {'Content-Type': 'application/json'} - this should be a part of a manager, util class.